prow/plugins.yaml

```diff
@@ -658,6 +658,7 @@ plugins:
   - trigger
 
   kubernetes-sigs/contributor-playground:
+  - invalidcommitmsg
```

Comment:
I thought it was best to try out the plugin on this repo first. Thoughts?

Also...does this need lazy consensus on the contribex ML?

Reply:
I don't think it needs lazy consensus, and I think this is a great place to use it :) This is a safe repo for folks to learn good habits and it helps with that. 👍

Reply:
No consensus needed on the contributor playground.